packages/modelviewer.dev/data/docs.json

```diff
@@ -552,6 +552,15 @@
         "links": [
           "<a href=\"../examples/scenegraph/#transforms\">Related example</a>"
         ]
+      },
+      {
+        "name": "interpolation-decay",
+        "htmlName": "interpolationDecay",
+        "description": "Controls model interaction sensitivity, the smaller the value the more sensitive the interaction will be.",
```

Comment:
How about: "Controls the rate of interpolation when the camera or model moves, due to either user interaction or attribute changes. The decay is asymptotic and the value is in milliseconds, where the majority of the movement will occur within this value's time. Doubling this value will cut the speed in half."

Reply:
Sure sounds great